pkg/security/probe/activity_dump_proto_enc_v1.go

```diff
@@ -261,13 +261,18 @@ func socketNodeToProto(sn *SocketNode) *adproto.SocketNode {
 		return nil
 	}
 
-	return &adproto.SocketNode{
+	psn := &adproto.SocketNode{
```

Comment:
Let's prealloc the slice like it's done for other fields

Reply:
👍🏻